js/oop.js

```diff
@@ -0,0 +1,115 @@
+var FoursquareRequest = function() {
+  this.searchCount = 0;
+};
+
+var Handlebars = function() {
+
+};
+var DomAction = function(){
+
+};
+var VenueLocation = function() {
+  var self = this;
+  var boxLat = $(self).data("lat");
+  var boxLng = $(self).data("lng");
+  var boxId = $(self).data("id");
+};
```

Comment:
This project is large enough that you may want to have a file for each class and its method definitions. Then the kickoff/glue code becomes much easier to follow, since that's the only thing remaining in the "main" file.